editor/js/editor-libs/events.js

```diff
@@ -24,23 +24,19 @@ function addCSSEditorEventListeners(exampleChoiceList) {
 
     exampleChoiceList.addEventListener('click', function(event) {
         var target = event.target;
-
-        // if the original target is not an `example-choice` element
-        if (!target.classList.contains('example-choice')) {
-            // find this element's `example-choice` parent
-            target = mceUtils.findParentChoiceElem(target);
-        }
-
-        if (target.classList.contains('copy')) {
-            mceAnalytics.trackEvent({
-                category: 'Interactive Example - CSS',
-                action: 'Copy to clipboard clicked',
-                label: 'Interaction Events'
-            });
+        var isClickedBetween = target.getAttribute('id') === 'example-choice-list';
```

Comment:
@tannerdolby I don’t think we need this anymore. We are only attaching events to the actual `example-choice` elements. We are no longer listening on the parent `example-choice-list`. Agree?

Reply:
Agreed, we don't need that stuff anymore as were only listening on the `.example-choice`'s themselves and not using `mce.findParentElem(event.target)` which made it necessary to return if `event.target` was the parent `example-choice-list`.

Reply:
Doh! I completely missed that it is still inside that event handler. We do not actually need that either anymore.

Reply:
A way to fix this, would be to simply place our `Array.from()` code outside of the `exampleChoiceList.addEventListener("click"..)` block so that the click event listeners get added to each choice right away instead of adding new event listeners every time the parent `example-choice-list` is clicked. This way, we only need to click each `example-choice` once as the click event is already attached and the callback `handleChoiceEvent` fires as expected.

Replace
```js
exampleChoiceList.addEventListener("click", () => {
  ... 
   Array.from().forEach((choice), => {
      choice.addEventListener("click", handleChoiceEvent)
   })
})
```

with 

```js
var exampleChoices = exampleChoiceList.querySelectorAll('.example-choice');
Array.from().forEach((choice), => {
      choice.addEventListener("click", handleChoiceEvent)
   })
```

Reply:
I fixed things up here, the click between choices still works as expected and doesn't produce any errors (or anything) for that matter as there isn't any click event attached to the parent `example-choice-list`. And now our logic is outside of the parent click event so things are firing as expected.

Reply:
I see that `var mceUtils = require('./mce-utils');` is unused in the imports so we could probably remove that as we aren't using any methods from `mceUtils` anymore.

Reply:
We can clean that up as well. Thanks, @tannerdolby

Reply:
Got it. Your welcome @schalkneethling !